sass/_layout.scss

```diff
@@ -141,6 +141,16 @@
 .has-margin {
     margin: $baseline !important;
 }
+.has-margin--tablet {
```

Comment:
the bulma convention is that if you specify a breakpoint name its that plus all the ones above - in this case tablet + desktop + widescreen + fullhd. Can you change this to better match? Its going to be confusing otherwise.